docs/concepts/copilot/enable-data-copilot.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Enable Data Copilot
+id: enable-data-copilot
+description: How to enable Prophecy Data Copilot for private VPC SaaS environments.
+sidebar_position: 10
+tags:
+  - concepts
+  - copilot
+  - installation
+  - upgrade
+---
+
+**Prophecy Data Copilot** is an AI-powered assistant that delivers intelligent suggestions and automates repetitive tasks for visual data transformations. You can read more about it at [Data Copilot](/concepts/copilot).
+
+Data Copilot leverages OpenAI's generative AI models to understand user intent, and enriched by the organizations' [knowledge graph](/concepts/copilot#knowledge-graph), to automate repetitive data engineering tasks. By default, Data Copilot leverages **Prophecy's managed OpenAI subscription and is entirely free** for existing Prophecy customers. Prophecy uses user queries and metadata when communicating with OpenAI. Prophecy never sends any customer data to OpenAI.
+
+However, for the most security conscious organizations, it is possible to configure Prophecy to use your own OpenAI endpoint. This page describes how to enable Prophecy Data Copilot for private VPC SaaS environments and configure it to use your own OpenAI or Azure OpenAI endpoint.
+
```

Comment:
New paragraph:

Prophecy also supports running Data Copilot from a locally hosted private model. Additionally, Prophecy Copilot can be configured to use custom LLM endpoints provided by AWS Bedrock, Google Vertex AI, and Databricks model serving.